site/en/develop/expression.md

```diff
@@ -0,0 +1,109 @@
+---
+id: expression.md
+title: Predicate Expressions
+---
+
+# Predicate Expressions
+A predicate is an expression that evaluates to a boolean value. Milvus conduct scalar filtering by searching with predicates. A predicate expression, when evaluated, returns either TRUE or FALSE.
```

Comment:
A predicate is an expression evaluated with a Boolean value. Milvus conducts scalar filtering by searching with predicates. A predicate expression, when evaluated, returns either TRUE or FALSE.